Calculator/Calculator/Models/Formula.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Formula.swift
+//  Calculator
+//
+//  Created by Charles on 2023/10/08.
+//
+
+struct Formula {
+    var operands: CalculatorItemQueue<Double>
+    var operators: CalculatorItemQueue<Operator>
+    
+    mutating func result() -> Double {
+        var accumulatingValue: Double = 0
```

Comment:
0을 하드코딩 하는 것 보다는 Double 타입에 있는 타입 프로퍼티를 활용하는 것을 추천드립니다. 👍
```suggestion
        var accumulatingValue: Double = .zero
```